internal/service/imagebuilder/distribution_configuration.go

```diff
@@ -168,6 +168,68 @@ func ResourceDistributionConfiguration() *schema.Resource {
 								},
 							},
 						},
+						"fast_launch_configuration": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							MaxItems: 1000,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"account_id": {
+										Type:         schema.TypeString,
+										Required:     true,
+										ValidateFunc: verify.ValidAccountID,
+									},
+									"enabled": {
+										Type:     schema.TypeBool,
+										Required: true,
+									},
+									"launch_template": {
+										Type:     schema.TypeList,
+										MaxItems: 1,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"launch_template_id": {
+													Type:         schema.TypeString,
+													Optional:     true,
+													ValidateFunc: verify.ValidLaunchTemplateID,
+												},
+												"launch_template_name": {
+													Type:         schema.TypeString,
+													Optional:     true,
+													ValidateFunc: verify.ValidLaunchTemplateName,
+												},
```

Comment:
Is there a way to use `ExactlyOneOf` in a nested set element? It cannot be done like below as the set can have more than 1 item.
```
ExactlyOneOf: []string{"distribution.0.fast_launch_configuration.0.launch_template.0.launch_template_id", ...}
```

Reply:
Unfortunately not right now (needs the great work being done over here: https://github.com/hashicorp/terraform-plugin-framework-validators) 😞.